log/field.go

```diff
@@ -152,6 +153,22 @@ func Lazy(ll LazyLogger) Field {
 	}
 }
 
+// Skip allows a field to be optionally be included in a Field list.
```

Comment:
I find the `error` example to be misleading, because it's not idiomatic to Go to do something with the error object without checking that it's not actually nil.  How about the following description:

```
// Skip creates a no-op log field that should be ignored by the tracer.
// It can be used to capture optional fields, for example those that should
// only be logged in non-production environment:
//
//     func cunstomerField(order *Order) log.Field {
//          if os.Getenv("ENVIRONMENT") == "dev" {
//              return log.Field("customer", order.Cusomer.ID)
//          }
//          return log.Skip()
//     }
//
//     span.LogFields(log.String("event", "purchase"), customerField(order))
//
```